.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -458,6 +458,14 @@ jobs:
           cmake --build . --config Release --parallel 4
           ./hello
 
+      - name: Build Sample [simple]
+        if: ${{ matrix.light == 'green' }}
+        run: |
+          ./build.sh -p host-interp
```

Comment:
Should `cd samples/simple` firstly?

Reply:
L467 `working-directory: ./samples/simple` will make sure of it